package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.25.0",
+  "version": "4.26.0",
```

Comment:
version 4.26.0 is good, but will need a rebase or merge from main since `p-icon--history` was merged today